src/styles/components/menu-slider.scss

```diff
@@ -0,0 +1,85 @@
+.menu {
+	&__slider {
+		position: fixed;
+		top: 0;
+		right: 0;
+		height: 100%;
+		width: 100%;
+		background-color: var(--color-text-secondary);
+		z-index: 2000;
+		padding-top: 3.4rem;
+		border-radius: 0.5rem;
+		overflow-x: hidden;
+		transition: 0.5s;
+	}
+
+	&__slider--active {
+		width: 0;
+	}
+
+	&__slider-item-container {
+		display: flex;
+		align-self: center;
+		justify-content: center;
+		flex-direction: column;
+		margin-top: 5rem;
+	}
+
+	&__slider-item-container {
+		display: flex;
+		align-self: center;
+		justify-content: center;
+		flex-direction: column;
+		margin-top: 5rem;
+	}
+
+	&__slider-item {
+		display: block;
+		text-decoration: none;
+		text-align: center;
+		margin: 2rem 2rem;
+		font-size: 2.5rem;
+		transition: 0.1s;
+
+		&:visited {
+			color: var(--color-text-primary);
+		}
+
+		&:hover {
+			color: var(--color-background-darker);
+			transition: 0.3s;
+		}
+	}
+
+	&__button-close {
+		display: flex;
+		justify-content: center;
+		position: absolute;
+		top: 1rem;
+		left: 1rem;
+		cursor: pointer;
+		width: 3rem;
+		height: 3rem;
+	}
+
+	&__button-cross {
+		pointer-events: none;
+	}
+
+	@include responsive('tablet') {
+		&__slider {
```

Comment:
move this to under `&__slider` to keep consistency with the other includes in the file

Reply:
Done